sm/sm.c

```diff
@@ -139,8 +139,6 @@ M0_INTERNAL void m0_sm_ast_post(struct m0_sm_group *grp, struct m0_sm_ast *ast)
 
 	do {
 		ast->sa_next = grp->s_forkq;
-		M0_LOG(M0_DEBUG, "grp=%p forkq_push: %p -> %p",
```

Comment:
Better comment these logs out, so it will be convenient to enable them back if needed to debug something.

Reply:
Commented the logs.